api/app/utils/daily_pdf_gen.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Mapping
+import pdfkit
+from jinja2 import Template
+
+from app.schemas.hfi_calc import FireCentre, HFIResultResponse, PlanningArea, WeatherStation
+from app.utils.pdf_data_formatter import response_2_daily_jinja_format
+
+daily_template_path = "api/app/utils/daily_template.html"
+daily_rendered_path = "api/app/utils/daily_rendered.html"
+
+output_file_path = "api/app/utils/out.pdf"
+
+
+def generate_daily_pdf(result: HFIResultResponse, fire_centres: List[FireCentre]):
```

Comment:
What is the return type? Not sure looking at code what pdfkit.from_file returns.

Reply:
Returns `True` if it succeeded, `False` otherwise. I'll add `bool` to the return signature.